specification/web/resource-manager/Microsoft.Web/2016-08-01/WebApps.json

```diff
@@ -13221,6 +16065,111 @@
         }
       }
     },
+    "FunctionEnvelope": {
+      "description": "Web Job Information.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyOnlyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "FunctionEnvelope resource specific properties",
+          "properties": {
+            "name": {
```

Comment:
@ Eric, mark the property with [ApiExplorerSettings(IgnoreApi = true)] in model. 

---
In reply to: [134406880](https://github.com/Azure/azure-rest-api-specs/pull/1551#discussion_r134406880) [](ancestors = 134406880)